ios/xcode/OpenGLES.xcodeproj/project.pbxproj

```diff
@@ -5638,6 +6247,7 @@
 				0A60525423464F51005CEA98 /* Products */,
 				0A60565F23465AED005CEA98 /* Frameworks */,
 				0A60525723464F51005CEA98 /* Info.plist */,
+				2F410AB524AA92CF001EB161 /* MetalANGLE_tvos copy-Info.plist */,
```

Comment:
Could you remove this `Info.plist` clone? And Also make all MetalANGLE.framework variant use `Info.plist`.
Whenever you duplicate `MetalANGLE` target, Xcode always creates a duplicated copy of `Info.plist` such as `MetalANGLE_tvos copy-Info.plist` and make the duplicated target use it which is not what we want.